mod/likes/views/default/likes/css.php

```diff
@@ -7,9 +7,8 @@
 /* ***************************************
 	Likes
 *************************************** */
-.elgg-likes {
+.elgg-likes-popup {
```

Comment:
Why was this changed? Doesn't this effectively change API if a theme was using the previous class?